mgmt/src/models/external/overlay/vpcpeering.rs

```diff
@@ -126,3 +126,150 @@ impl VpcPeeringTable {
             .filter(move |p| p.left.name == vpc || p.right.name == vpc)
     }
 }
+
+#[cfg(test)]
```

Comment:
@qmonnet, I believe that some of the tests added here overlap with existing tests in https://github.com/githedgehog/dataplane/blob/pr/qmonnet/nat-validation/mgmt/src/models/external/overlay/tests.rs 
Did you double-check?

Reply:
There's some overlap in that we're testing closely related components, but in `test.rs` we test things build, in `vpcpeering.rs` we test they're valid - that's the distinction we talked about before. So I'm not sure the tests are plain redundant. `test_manifest_expose()` added here looks closely to the tests for `add_expose()` in `test.rs`, but it also checks the result of `get_peering_manifests()` at the end. It also seems that all cases returning errors in `test.rs` or `vpcpeering.rs` are distinct.

Reply:
Hum. I believe that in test.rs we also test things like:
  * duplicated vpc names
  * that peerings refer to existing VPCs.
  * we test Overlay::validate()
  * the filtered lookups to find the peerings that a VPC participates in.

In any case, can you move the tests there so that they are in a single place and we can see what the tests cover altogether. The reason to add a separate module is not to bloat the functional code with large tests.

Reply:
Thanks!